src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java

```diff
@@ -611,6 +621,7 @@ private void testAllLocalesAndPages() {
 				}
 			}
 		}
+		*/
```

Comment:
please excuse the commented-out code here -- I'll follow up with another PR to clean it up